### DISCUSSION TITLE
Filter Skeleton Detection

### DISCUSSION THREAD

**Discussion by AngyIce** [endink/Mediapipe4u-plugin#138] [Q&A] [answered]
Hello there, is there a way to prevent media pipe detection of non-human bodies during the simulation ?? I've noticed that sometimes some objects (or groups of objects) trigger the skeleton even if they are not human. Of course only if there's no one in front of the camera. Thank you

**Comment by endink** [comment, marked as answer]
Sorry, its not easy. You can use the body pitch to assert whether it's a person or not, and you can also suppress low confidence by using the `MinPoseScoreThresh` property on `MediaPipeAnimaInstance` (your animation blueprint) provided by **MediaPipe4U**  to suppress low confidence and not react
